test/e2e/handler/rollback_test.go

```diff
@@ -93,18 +92,12 @@ func discoverNameServers(nic string) nmstate.State {
 
 var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:component]rollback", func() {
 	Context("when an error happens during state configuration", func() {
-		BeforeEach(func() {
-			By("Rename vlan-filtering to vlan-filtering.bak to force failure during state configuration")
-			runner.RunAtHandlerPods("mv", "/usr/local/bin/vlan-filtering", "/usr/local/bin/vlan-filtering.bak")
-		})
 		AfterEach(func() {
-			By("Rename vlan-filtering.bak to vlan-filtering to leave it as it was")
-			runner.RunAtHandlerPods("mv", "/usr/local/bin/vlan-filtering.bak", "/usr/local/bin/vlan-filtering")
 			updateDesiredStateAndWait(linuxBrAbsent(bridge1))
 			resetDesiredStateForNodes()
 		})
 		It("should rollback failed state configuration", func() {
-			updateDesiredState(linuxBrUpNoPorts(bridge1))
+			updateDesiredState(linuxBrBadYaml(bridge1))
```

Comment:
This is not the same as the previous test. The previous one rolled back after the configuration was applied on the system. With this change, it never attempts to change the system. You don't have to fix it in the cherry-pick, but it would be great to have a follow-up for this.

Reply:
You are right